ts/state/ducks/conversations.ts

```diff
@@ -71,7 +75,17 @@ export type FindAndFormatContactType = {
   isMe: boolean;
 };
 
-export type PropsForExpirationTimer = {
+export type PropsForExpiringMessage = {
+  convoId?: string;
+  messageId: string;
+  direction: MessageModelType;
+  expirationTimestamp?: number | null;
+  expirationLength?: number | null;
+  isExpired?: boolean;
+};
+
+export interface PropsForExpirationTimer extends PropsForExpiringMessage {
```

Comment:
you can also just use the & operator on type rather than making everything interfaces extending one another

Reply:
Nice catch and good point. I think I just use interface out of habit because that is how I first learned typescript inheritance. But using type + & is more consistent with the code base and better based on my mental model of typescript now thanks to type-level typescript. Fixed